required/latex-lab/latex-lab-toc-kernel-changes.dtx

```diff
@@ -291,10 +322,11 @@
      \csname contentsline@text@#1@format\endcsname{#4}%
      \UseHookWithArguments{contentsline/text/after}{4}{#1}{#4}{#5}{\@contentsline@destination}%
      \nobreak
-     \@dottedtocline@cfgpoint@leaders{%
+     \UseTaggingSocket{toc/leaders/before}\SuspendTagging{toc/leaders}%
```

Comment:
I think that the \SuspendTagging and \ResumeTagging here is actually something which is part of the plug implementation and shouldn't show up here on this level.

Reply:
You can't put the \ResumeTagging into the plug, it is then not seen if \SuspendTagging is active as the tagging socket is then ignored. One could put the \SuspendTagging but it wouldn't be symmetric.

Reply:
hmm, yes, can't argue against that.